content/en/building/guides/interoperability/cht_config.md

```diff
@@ -1,12 +1,13 @@
 ---
```

Comment:
Please replace `_` with `-` in the file name to ensure consistency across the repo.